Code/Legacy/CryCommon/ISystem.h

```diff
@@ -1426,6 +1426,7 @@ namespace Detail
 #define CryLog(...) ((void)0)
 #define CryComment(...) ((void)0)
 #define CryLogAlways(...) ((void)0)
+#define CryOutputToCallback(...) ((void)0)
```

Comment:
just curious will all of this be removed at some pointer?

Reply:
Hopefully some point in the future it would be removed